src/components/editor/yaml-metadata/yaml-metadata.ts

```diff
@@ -53,7 +54,16 @@ export class YAMLMetaData {
       transition: 'none',
       theme: 'white'
     } */
-    this.tags = rawData?.tags?.split(',').map(entry => entry.trim()) ?? []
+    if (typeof rawData?.tags === 'string') {
+      this.tags = rawData?.tags?.split(',').map(entry => entry.trim()) ?? []
+      this.deprecatedTagsSyntax = true
+    } else if (typeof rawData?.tags === 'object') {
+      this.tags = rawData?.tags?.filter(tag => tag !== null) ?? []
+      this.deprecatedTagsSyntax = false
+    } else {
+      this.tags = []
+      this.deprecatedTagsSyntax = true
```

Comment:
Wouldn't this show the error if there are no tags?

Reply:
No this would be called if a user would put something besides a string or a string list into tags.

Reply:
Are you sure?.. because if it's not defined then it's neither a string of an array. 

I can't test it right now.

Reply:
You were right
![Bildschirmfoto von 2020-12-23 12-46-49](https://user-images.githubusercontent.com/3978662/102993180-fbc00b80-451c-11eb-9a71-570f401772de.png)

Reply:
```suggestion
      this.deprecatedTagsSyntax = false
```
should fix this.

Reply:
Done